webpack/ForemanInventoryUpload/Components/PageHeader/components/DocsButton/DocsButton.js

```diff
@@ -2,11 +2,9 @@ import React from 'react';
 import { Button, Icon } from 'patternfly-react';
 import { DOCS_BUTTON_TEXT } from '../../../../ForemanInventoryConstants';
 import { getInventoryDocsUrl } from '../../../../ForemanInventoryHelpers';
-import './docsButton.scss';
 
-const DocsButton = () => (
+const ToolbarButtons = () => (
```

Comment:
It exports only the `DocsButton`, isn't it?

Reply:
yes, it was a leftover from before splitting the component, fixed!